helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -90,33 +92,98 @@
 
 
 public class ZKHelixAdmin implements HelixAdmin {
+  private static final Logger LOG = LoggerFactory.getLogger(ZKHelixAdmin.class);
+
+
```

Comment:
Nit: one empty line would suffice?

Reply:
How could 2 empty lines be added...
Addressed.